cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,127 @@
 describe('Web Tables page', () => {
-  it('', () => {
+
+  const user = {
+    firstName: 'John',
+    lastName: 'Doe',
+    Email: 'JohnDoe@qa.team',
+    age: '36',
+    salary: '1000',
+    department: 'Test'
+  }
+
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables')
+  });
+
+  it('should have pagination', () => {
+    cy.get('.-pagination')
+      .should('contain', 'Previous')
+      .and('contain', 'Next');
+    cy.contains('.-pageInfo', 'Page')
+      .should('exist');
+  });
+
+  it('should have row count selection', () => {
```

Comment:
Add an assertion to check if the row count changes to 5 after selecting '5 rows' from the dropdown

Reply:
Done